docs/reference/schemas/resource/manifest/root.md

```diff
@@ -229,11 +229,26 @@ information, see the [DSC Resource manifest provider property schema reference][
 The `exitCodes` property defines a set of valid exit codes for the resource and their meaning.
 Define this property as a set of key-value pairs where:
 
-- The key is a string containing an integer that maps to a known exit code for the resource.
+- The key is a string containing an integer that maps to a known exit code for the resource. You
```

Comment:
Should we just say this is a signed int?

Reply:
Don't see this change yet?

Reply:
The change is here for me:

https://github.com/PowerShell/DSC/blob/916548c74fcef82ec3f3e84cf917d969f9d826ad/docs/reference/schemas/resource/manifest/root.md?plain=1#L232-L235